packages/cozy-doctypes/src/contacts/Contact.js

```diff
@@ -9,6 +9,12 @@ const getPrimaryOrFirst = property => obj => {
   return obj[property].find(property => property.primary) || obj[property][0]
 }
 
+const isDeprecated = methodName =>
```

Comment:
Super minor nit, but generally `isSomething` is a good way to identify that a variable is a boolean. I think for a function, `logDeprecated` or some name that implies and action makes more sense.

Reply:
I had the same though but no better idea, thanks